framework/source/class/qx/bom/AnimationFrame.js

```diff
@@ -76,11 +76,13 @@ qx.Bootstrap.define("qx.bom.AnimationFrame",
      * soon as the given duration is over.
      *
      * @param duration {Number} The duration the sequence should take.
+     *
+     * @ignore(performance.*)
      */
     startSequence : function(duration) {
       this.__canceled = false;
 
-      var start = +(new Date());
+      var start = (window.performance && window.performance.now) ? (performance.now() + qx.bom.AnimationFrame.__start) : Date.now();
```

Comment:
Hmm. I see that the original code uses the check window.performance in the defer section, so it should be safe then.

@peuter do you think the  qx.core.Environment.get("performance.timer") is worth it?